docs/editors/vim.md

```diff
@@ -284,23 +296,62 @@ To close a buffer and return to the previous buffer, run the following command.
 
 This command is helpful when navigating in library dependency sources in the .metals/readonly directory.
 
-### Shut down the language server
+## Shut down the language server
 
 The Metals server is shutdown when you exit vim as usual.
 
 ```vim
 :wq
 ```
 
-This step clean ups resources that are used by the server.
+## Statusline integration
+
+`coc.nvim` has multiple ways to integrate with various statusline plugins. You can find instructions
+for each of them located [here](https://github.com/neoclide/coc.nvim/wiki/Statusline-integration).
+Two noteworthy things that they add are the ability to see diagnostic information in the current
+buffer...
```

Comment:
What do the triple dots mean?
```suggestion
buffer.
```

Reply:
Aha, it seems to be to imply the sentence continues. I generally try to avoid letting sentences go over images since it's difficult to read the sentence that way

Reply:
Sounds good. I'll change this around to not have these.